catalog/spatial/registry/registry-ebrim-transformer/src/main/java/org/codice/ddf/registry/transformer/RegistryTransformer.java

```diff
@@ -115,8 +116,7 @@ public Metacard transform(InputStream inputStream, String id)
     @Override
     public BinaryContent transform(Metacard metacard, Map<String, Serializable> arguments)
             throws CatalogTransformerException {
-        if (metacard.getTags()
-                .contains(RegistryConstants.REGISTRY_TAG)) {
+        if (RegistryUtility.hasRegistryId(metacard)) {
```

Comment:
This is where a isRegistryMetacard function would be nice. It would do the tags and registry id checks